src/stores/torrent-store.js

```diff
@@ -22,13 +22,11 @@ const extractDomains = (torrent) => {
 class TorrentStore {
   @observable previousTorrents = [];
   @observable torrents = [];
-  @observable statusFilter = 0;
   @observable trackerFilter = '';
   @observable textFilter = '';
-  @observable sortCriteria = 'name';
-  @observable sortDirection = '';
 
-  constructor(rpc) {
+  constructor(rpc, prefs_store) {
```

Comment:
Scratch that, I realized I can't actually do that since the computed property is being accessed through numerous sources.

While coupling is less than ideal here I'm not sure what we could do. We can just let the components aggregate the data directly but it's non-ideal too IMO as the computed decorator probably helps.

I just checked and it seems like coupling is the accepted solution for this use case in mobx:
https://github.com/mobxjs/mobx/issues/223

Eager to hear if you think there's a better way @edulan!